pkg/actor/generate_cert.go

```diff
@@ -175,6 +189,9 @@ func (rc *generateCert) generateCA(ctx context.Context, log logr.Logger, cluster
 	// if the secret is ready then don't update the secret
 	// the Actor should have already generated the secret
 	if secret.ReadyCA() {
+		if err := ioutil.WriteFile(rc.CAKey, secret.CAKey(), 0600); err != nil {
```

Comment:
Was this just missing before but needed for something?

Reply:
We need to regenerate the node certificates when SQLhost is added/updated, but to regenerate using `cockroach` CLI, CA key and cert is required which are updated in the secret. So here we are writing those key and cert inside the controller on a temp path for node certificate regeneration